<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывод:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Интенсивность поступления заявок:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_model">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Смоделировать СМО</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Интенсивность обработки заявок:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Разброс интенсивности обработки заявок:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время моделирования:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_intensivity_gen">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_intensivity_oa">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_intensivity_oa_range">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99999.990000000005239</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_time_model">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>501</width>
      <height>192</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Загруженность системы</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Среднее время пребывания завки в СМО</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Обработанные заявки</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Общее время моделирования</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Теоретич.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фактич.</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
